ckan_cloud_operator/providers/ckan/env/manager.py

```diff
@@ -22,373 +21,159 @@
-            try:
-                spec = instance_data['spec']
-                for k in spec_keys:
-                    instance_data[k] = spec.get(k)
-                metadata = instance_data['metadata']
-                for k in metadata_keys:
-                    instance_data[k] = metadata.get(k)
-                instance_type = instance_data['metadata']['labels'].get('{}/instance-type'.format(labels_manager.get_label_prefix()))
-                deployment = deployment_manager.get(instance_data['id'], instance_type, instance_data)
-                instance_data['ready'] = deployment.get('ready')
-            except Exception as e:
-                pass
-            if not full:
-                instance_data = dict(
-                    (k, v)
-                    for k, v in instance_data.items()
-                    if k in ('ready', *spec_keys, *metadata_keys)
-                )
-            if withCredentials:
-                instance_data['admin_password'] = config_manager.get(
-                    'CKAN_ADMIN_PASSWORD',
-                    secret_name='ckan-admin-password',
-                    namespace=instance_data['id']
-                )
-            yield instance_data
-
-
-def delete_name(instance_name):
-    crds_manager.delete(INSTANCE_NAME_CRD_SINGULAR, instance_name)
-
-
-def set_name(instance_id, instance_name, dry_run=False):
-    resource = crds_manager.get(INSTANCE_NAME_CRD_SINGULAR, name=instance_name, required=False)
-    if resource:
-        resource['spec']['latest-instance-id'] = instance_id
-        if not resource['spec']['instance-ids'].get(instance_id):
-            resource['spec']['instance-ids'][instance_id] = {'added': datetime.datetime.now()}
-    else:
-        resource = crds_manager.get_resource(
-            INSTANCE_NAME_CRD_SINGULAR, instance_name,
-            spec={
-                'name': instance_name,
-                'latest-instance-id': instance_id,
-                'instance-ids': {
-                    instance_id: {'added': datetime.datetime.now()}
-                }
-            }
-        )
-    if dry_run:
-        logs.print_yaml_dump(resource)
-    else:
-        kubectl.apply(resource)
 
+def remove_environment(environment):
+    updated_envs = []
+    cco_config = {}
+    if _cco_is_configured() and _get_config():
+        cco_config = _get_config()
+        env_exists = any([environment.lower() == _env.get('name', '').lower() for _env in cco_config.get('environments') or []])
+        if not env_exists:
+            logs.info(f'Environment {environment} does not exists please run `cco ckan env add {environment}`')
+            return
+        envs = cco_config.get('environments', [])
 
-def set_storage(instance_id, instance_name, dry_run=False):
-    from ckan_cloud_operator.providers.storage.manager import get_provider, get_provider_id
+        for _env in envs:
+            if _env.get('name').lower() == environment.lower():
+                continue
```

Comment:
Is this continue intentional? Or it should be removing environments?

Reply:
Intentional - this is for removing environments. It collects all but selected